plugins/repositories/core/src/main/javascript/app/components/shared/failure/failure.css

```diff
@@ -1,6 +1,69 @@
 .fail {
   background-image: url("../../../img/fail.png");
-  width: 67px;
-  height: 58px;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 65px;
+  height: 60px;
   margin: auto;
+  padding: 50px;
 }
+
+#repository-manager-failure-header {
```

Comment:
Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.